spec/rubocop/cop/style/sole_nested_conditional_spec.rb

```diff
@@ -392,7 +427,7 @@ def foo
     RUBY
 
     expect_correction(<<~RUBY)
-      if foo.is_a?(Foo) && (bar && baz) && quux
+      if foo.is_a?(Foo) && bar && baz && quux
```

Comment:
I can bring these parentheses back if you want me to, but they're not actually needed logic-wise. They're also not removed by `Style/RedundantParentheses` (should that cop autocorrect such cases?), so I think it's better that this cop doesn't add them.